src/window/building_info.c

```diff
@@ -453,7 +455,8 @@ static void init(int grid_offset)
         case 5: context.height_blocks = 24; break;
         case 6: context.height_blocks = 38; break;
         case 7: context.height_blocks = 26; break;
-        case 8: context.height_blocks = 40; context.width_blocks = 30; break;
+        case 8: context.height_blocks = 40; context.width_blocks = 30; break;        
+        case 10: context.height_blocks = 47; context.width_blocks = 30; break;
         case 9: context.height_blocks = 20; break;
```

Comment:
Extra tab at the end of the `case 8:` line

`case 9:` and `case 10:` lines swapped.


```suggestion
        case 8: context.height_blocks = 40; context.width_blocks = 30; break;
        case 9: context.height_blocks = 20; break;
        case 10: context.height_blocks = 47; context.width_blocks = 30; break;
```